### PR TITLE
Refs 309: Add/Update ouiaIds for popular/custom tables

### PR DESCRIPTION
**Pull request by mshriver**
Updated an aria label as well to be accurate.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-309

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/Routes.tsx [hunk 1]
```diff
@@ -69,8 +69,8 @@ export default function MainRoutes() {
   return (
     <>
       <PageHeader>
-        <PageHeaderTitle title='Custom Repositories' />
-        <Text className={classes.subtext}>
+        <PageHeaderTitle title='Custom Repositories' ouiaId='custom_repositories_title'/>
```
Comment (mshriver) [inline src/Routes.tsx:72]: This one isn't working :rage1:

Comment (mshriver) [reply src/Routes.tsx]: Not a problem in the short term, but something Andrew and I talked about coming back to.

**Pull request merged**